django/contrib/postgres/indexes.py

```diff
@@ -1,8 +1,26 @@
 from __future__ import unicode_literals
 
-from django.db.models import Index
+from django.db.models.indexes import Index
 
-__all__ = ['GinIndex']
+__all__ = ['BRINIndex', 'GinIndex']
+
+
+class BRINIndex(Index):
+    """
+    Support for the BRIN index introduced in PostgreSQL 9.5. There's an optional
```

Comment:
I don't think it's important to mention PostgreSQL version details in the docstring.